bundles/org.openhab.binding.chatgpt/src/main/java/org/openhab/binding/chatgpt/internal/dto/ChatResponse.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.chatgpt.internal.dto;
+
+import java.util.List;
+
+/**
+ * This is a dto used for parsing the JSON response from ChatGPT.
+ *
+ * @author Kai Kreuzer - Initial contribution
+ *
+ */
+public class ChatResponse {
+
+    private List<Choice> choices;
+    private String id;
+    private String object;
+    private int created;
+    private String model;
+
+    public List<Choice> getChoices() {
+        return choices;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public int getCreated() {
+        return created;
+    }
+
+    public String getObject() {
+        return object;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public static class Choice {
+        private Message message;
+        private String finish_reason;
```

Comment:
You can fix this SAT naming issue by using `@SerializedName(finish_reason")` or creating a `Gson` instance using:

```java
new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES).create()
```